schemas/context/enduserids.example.1.json

```diff
@@ -2,8 +2,8 @@
   "https://ns.adobe.com/experience/mcid": {
     "@id": "https://ns.adobe.com/entities/identity/mcid123",
     "xdm:namespace": {
-      "@id": "https://ns.adobe.com/entities/namespace/11111",
-      "xdm:code": "AA11111"
+      "@id": "https://ns.adobe.com/entities/namespace/AVID",
+      "xdm:code": "4"
```

Comment:
Is a longer value more realistic?

Reply:
I had a good review with the Identity team right after this which gave me good information to help make this sort of example more illustrative. I will update accordingly.

Reply:
Ok, then that's the last remaining comment that needs to be addressed. Please also resolve the conflict.